spark-cassandra-connector/src/test/scala/com/datastax/spark/connector/util/ReflectionUtilSpec.scala

```diff
@@ -11,6 +14,25 @@ class ReflectionUtilSpec extends FlatSpec with Matchers {
     factory should be(DefaultConnectionFactory)
   }
 
+  it should "be able to find DefaultConnectionFactory in a multi-threaded context" in {
+    val conductor = new Conductor
+    import conductor._
+
+    thread("ThreadA") {
```

Comment:
Ok, so we need to pick a different class, because the one  you used was not present in 1.1.